app/presenters/organisations/documents_presenter.rb

```diff
@@ -46,21 +46,26 @@ def has_latest_documents?
     end
 
     def latest_documents
-      @latest_documents ||= Services.cached_search(
+      @latest_documents ||= latest_documents_excluding_org_page
+      search_results_to_documents(@latest_documents, @org)
+    end
+
+  private
+
+    def latest_documents_excluding_org_page
+      documents = Services.cached_search(
         {
-          count: 3,
+          count: 4,
           order: "-public_timestamp",
           filter_organisations: @org.slug,
           fields: %w[title link content_store_document_type public_timestamp],
         },
         metric_key: "organisations.search.request_time",
       )["results"]
 
-      search_results_to_documents(@latest_documents, @org)
+      documents.reject { |doc| doc["link"] == @org.base_path }.take(3)
```

Comment:
Do we need to add an additional step to make sure we're selecting the 3 most recent? eg if 4 documents are returned, and 0 of those documents are the organisation page itself.